web/locales/en/plugin__netobserv-plugin.json

```diff
@@ -117,8 +117,9 @@
   "Swap source and destination filters": "Swap source and destination filters",
   "Back and forth": "Back and forth",
   "One way": "One way",
-  "Exclude return traffic": "Exclude return traffic",
-  "Include return traffic, with swapped source and destination filters": "Include return traffic, with swapped source and destination filters",
+  "Switch between one way / back and forth filtering": "Switch between one way / back and forth filtering",
+  "One way will only show Source or Destination traffic according to your filters": "One way will only show Source or Destination traffic according to your filters",
```

Comment:
```suggestion
  "One way only shows Source or Destination traffic according to your filters": "One way only shows Source or Destination traffic according to your filters",
```

Reply:
I will follow that conversation instead
https://redhat-internal.slack.com/archives/C02939DP5L5/p1693212771687359?thread_ts=1692819555.340829&cid=C02939DP5L5

Reply:
I can still get rid of "will show" :smile: